src/analysis/complex/basic.lean

```diff
@@ -189,3 +190,65 @@ by simp [is_R_or_C.norm_sq, complex.norm_sq]
 by simp [is_R_or_C.abs, complex.abs]
 
 end is_R_or_C
+
+section complex_fderiv_properties
```

Comment:
Would this stuff maybe make more sense in `analysis.complex.real_deriv`? Another reviewer should probably make the call since this isn't my area.

Reply:
I am not sure. I have the impression that `analysis.complex.real_deriv` is specifically for the real restriction of a holomorphic function.

Reply:
I agree with Eric, the idea of splitting the two files was to not have the differentiability library imported in `analysis.complex.basic`.

Reply:
> I agree with Eric, the idea of splitting the two files was to not have the differentiability library imported in `analysis.complex.basic`.

Should I put the lemmas into `analysis.complex.real_fderiv`? Or is it better to create a new file?

Reply:
The former, I would say!